lib/Icon/index.js

```diff
@@ -168,34 +172,47 @@ const getIcon = name => {
     embed,
     people: peopleSVG,
     item: itemSVG,
+    boldTickSmall: boldTickSmallSVG,
+    approved: approvedSVG,
     question: questionSVG,
     hierarchy: hierarchySVG,
     userInvite: userInviteSVG
   };
   return icons[name] ? icons[name] : () => null;
 };
 
+const createClassesFromTypesList = (types, prefix = 'icon--') =>
+  types.reduce((className, type) => `${className} ${prefix}${type}`, '');
+
```

Comment:
Goes without saying, but make sure you catch all uses of this on the app!

Reply:
Actually I forgot to do this, so probably doesn't go without saying! 😆